APLSource/Tests/MkUniqueDir.aplf

```diff
@@ -0,0 +1,11 @@
+ {r}←MkUniqueDir w
+⍝ emulates ⎕MKDIR⍠'Unique' in versions < 20
+
+ :If 20≤2 1⊃'.'⎕VFI 2⊃'#'⎕WG'APLVersion'
```

Comment:
Nit: This can be moved to a utils file which has all of these helper functions, helps code to be more readable.

Something as simple as 
```
version←...
```

Reply:
I am not sure what impact that will have. @cursork can you please comment on this when you are looking at the PR?

Reply:
I even tried to take some time away to think... I agree that making it part of the API would also give me reason to pause for the exact reason @mbaas2 mentions. On the other hand, it's very unambiguous on what it means if added to the API.

I think it'd come down to if we'd find ourselves using it more or not in the future? Or if we expect to...